src/core/ext/transport/chttp2/transport/internal.h

```diff
@@ -759,13 +763,20 @@ struct grpc_chttp2_stream {
 void grpc_chttp2_initiate_write(grpc_chttp2_transport* t,
                                 grpc_chttp2_initiate_write_reason reason);
 
+struct TcpCallTracerWrapper {
```

Comment:
I suggest something more descriptive than "Wrapper", which says nothing about the type. How about `OffsetTcpCallTracer`?

Reply:
done